src/applications/ask-va/config/chapters/yourQuestion/questionAbout.js

```diff
@@ -22,6 +22,9 @@ const questionAboutPage = {
         labels: questionAboutLabels,
         descriptions: questionAboutDescriptions,
         required: () => true,
+        errorMessages: {
```

Comment:
Should we go ahead and add the Sign in portion back in to match Figma since we're done with form testing for now?

Reply:
missed that thanks